internal/nodes/avi_model_l4_translator.go

```diff
@@ -451,14 +451,113 @@ func PopulateServersForNodePort(poolNode *AviPoolNode, ns string, serviceName st
 				}
 
 			}
+			node4Found := true
+			node6Found := true
```

Comment:
One Generic comment: Can we make this logic as a separate function? Reason: It will reduce overall length of parent function and easier to go through it without scrolling up and down.